### DISCUSSION TITLE
Does it make sense to use fail2ban together with a reverseproxy?

### DISCUSSION THREAD

**Discussion by sjjh** [nextcloud/all-in-one#5010] [Questions] [answered]
I have a comprehension question: In my setup I'm going to run a reverse proxy on a separate host than nc-aio. Does it make sense to use the community-containers/fail2ban container, or should filtering happen on the reverse proxy? Thanks for any insights. :)

**Comment by szaimen** [comment, marked as answer]
> Does it make sense to use the community-containers/fail2ban container,

yes, it does :)
